lib/components/formik/ActionBlock.js

```diff
@@ -1,23 +1,28 @@
-import React, { useEffect } from "react";
+import React from "react";
 import Button from "../Button";
 import { useFormikContext } from "formik";
 
-const ActionBlock = () => {
+
+const ActionBlock = props => {
   const { handleReset, handleSubmit, isSubmitting } = useFormikContext();
+
   return (
     <div className="flex flex-row items-center justify-end">
       <Button
         style="secondary"
         label="Reset changes"
         className="mr-4"
         onClick={handleReset}
+        dataTestId={props["reset-data-test-id"] || "reset-changes-button"}
```

Comment:
We should be passing it as `data-test-id`

Reply:
@goutham-subramanyam _a

So what about the `data-test-id` for next save Button? Can both the buttons have the same `data-test-id` value? In neeto-help it's not like that. Thus two different prop keys.

Please confirm whether you want same `data-test-id` for both the Buttons. Thanks

Reply:
@yedhink 

```suggestion
        data-test-id={props["reset-data-test-id"] || "reset-changes-button"}
```